public/restaurant.js

```diff
@@ -0,0 +1,69 @@
+//Set Up Leaflet Map, centered on Burlington VT
+let myMap = L.map('restaurant-map').setView([44.4774, -73.2121], 16);
```

Comment:
You may want to set your zoom level higher when you go to the single restaurant page as a UX note.